<html><body><p>The new traceback is: <br><br>Traceback (most recent call last):<br>  File "/home/chshrcat/build/pypy-trunk/pypy/translator/c/gcc/trackgcroot.py", line 1211, in &lt;module&gt;<br>    assert fn.endswith('.s')<br>AssertionError<br><br>Is the position in input tracked? that might help, or I could package my .gcmap files.</p></body></html>